sklearn/linear_model/sag_fast.pyx

```diff
@@ -225,13 +229,15 @@ def sag(SequentialDataset dataset,
         str loss_function,
         double step_size,
         double alpha,
+        double beta,
         np.ndarray[double, ndim=2, mode='c'] sum_gradient_init,
         np.ndarray[double, ndim=2, mode='c'] gradient_memory_init,
         np.ndarray[bint, ndim=1, mode='c'] seen_init,
         int num_seen,
         bint fit_intercept,
         np.ndarray[double, ndim=1, mode='c'] intercept_sum_gradient_init,
         double intercept_decay,
+        bint saga,
         bint verbose):
     """Stochastic Average Gradient (SAG) solver.
```

Comment:
Please update this docstring to make it explicit that this function implements both SAG and SAGA.